vault/hcp_link/proto/meta/meta.proto

```diff
@@ -34,6 +34,57 @@ message ListAuthResponse {
   repeated Auth Auths = 1;
 }
 
+message GetClusterStatusRequest {}
+
+message HANode {
+  string Hostname = 1;
+}
+
+message RaftServer {
+  // NodeID is the name of the server
+  string NodeID = 1;
+
+  // Address is the IP:port of the server, used for Raft communications
+  string Address = 2;
+
+  // Leader is true if this server is the current cluster leader
+  bool Leader = 3;
+
+  // Protocol version is the raft protocol version used by the server
+  string ProtocolVersion = 4;
+
+  // Voter is true if this server has a vote in the cluster. This might
+  // be false if the server is staging and still coming online.
+  bool Voter = 5;
+}
+
+message RaftConfiguration {
+  repeated RaftServer Servers = 1;
+}
+
+message AutopilotServer {
+  string ID = 1;
+  bool Healthy = 2;
+}
+
+message AutopilotStatus {
+  bool Healthy = 1;
+  repeated AutopilotServer Servers = 2;
+}
```

Comment:
So the other Raft server information is meant to provide the Raft configuration whereas this is the Autopilot status. The data is actually accessible in different ways today, [sys/storage/raft/configuration](https://developer.hashicorp.com/vault/api-docs/system/storage/raft#read-raft-configuration) versus [sys/storage/raft/autopilot/state](https://developer.hashicorp.com/vault/api-docs/system/storage/raftautopilot#get-cluster-state) which is why I kept them separate.

They might be a bit redundant now but there is additional autopilot information that could be provided in the future apart from the general configuration like time since last contact, failure tolerance, redundancy zones, and more which we could potentially use in the future.

Reply:
This is good context. Thanks